### ISSUE TITLE
Project: TPW Project Management

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#2784]
An effort to standardize the way ATD manages projects and tasks.

## Background & Context

This project encompasses a variety of efforts to standardize the way ATD manages projects and tasks. In this context, a "project" may refer to infrastructure projects, programs, campaigns, and performance measurement. Although the [Moped](https://github.com/cityofaustin/atd-data-tech/issues/307) is designed to track all mobility infrastructure projects, this project will evaluate the practices and technologies all ATD staff use to manage their work, and seeksto identify ways that we can increase visibility of our activities at multiple scales (i.e., workgroup, division, and department levels).

## Outcomes & Deliverables

- division-level capacity-building in task and project management
- a dept-wide project tracking solution that  provides a birds-eye view of ATD's portfolio of work
- the development of a data warehouse which makes performance data available in a central location
- integrations across existing project management tools (Moped, Wryke, Github, ...)

### Resources
- [6/18 presentation to the Executive Team presentation](https://docs.google.com/presentation/d/1DgN525GtPcvaHChqzM1pRVKqOdV6Al2Nbp0Po-CWMV4/edit#slide=id.g85a8a163f6_0_21)
- [Drive folder](https://drive.google.com/drive/u/2/folders/1CcHYv042pY72dSmweyYFgF4ZuV8kEEky)

---
This Github issue represents a project of Austin Transportation's [Data & Technology Services](https://data.mobility.austin.gov/about/) team. Project status is documented regularly in the comments below.

**Comment by amenity** [comment]
Thoughts: Do we want to email division managers to learn what solutions they are using or have tried? Or create a brief survey?

**Comment by johnclary** [comment]
Definitely—let's include our research plans in the proposal, and we'll need to stress hard that we want to accommodate existing solutions as much as possible.

**Comment by johnclary** [comment]
Will probably me on the 6/16 e-team agenda. @amenity will forward you the invite when I have it but you can you block off your morning?

**Comment by johnclary** [comment]
[We presented](https://github.com/cityofaustin/atd-data-tech/issues/2786) to the e-team. Rob is going to think on our findings/proposal and reach out again. In the meantime, we agreed that focusing on Moped would have the biggest impact in advancing Rob's goal for better visibility into ATD's work.

@amenity I'm relegating this to the backlog for the time being.

**Comment by amenity** [comment]
Lauren S. just asked for advice on project management software and it reminded me of this outstanding request. Have you heard anything about it, @johnclary ? 

I'd be really interested to learn what people are using now that teams are more settled into remote work. I'd also be interested to see what using Microsoft Projects looks like in practice at the City (with our specific enterprise subscription and configuration)...

**Comment by johnclary** [comment]
@amenity no, i think it's iceboxed until Moped is off the ground. i think our [recommendation](https://docs.google.com/presentation/d/1DgN525GtPcvaHChqzM1pRVKqOdV6Al2Nbp0Po-CWMV4/edit#slide=id.g85a8a163f6_0_21) to do a research + consulting project around project management was a really good idea, but we made it clear we can't do it until it's resourced. 

we can definitely connect Lauren to people (Laura D) and platforms (trello, wryke, Monday, ...). not sure how else we can be of help.

**Comment by amenity** [comment]
Repy to Lauren's email may be useful here:

**Comment by amenity** [comment]
Note: Lauren and others at SMB are now using OneNote for lightweight team coordination and planning. Also, CTM now has training modules available:
- [OneNote 101](https://cityofaustin.sharepoint.com/sites/CTMLMS365/CTM-O365BootCamp-OneNote101/SitePages/Course.aspx) (2 hour class) – Mar 3rd
- [Planner 101](https://cityofaustin.sharepoint.com/sites/CTMLMS365/CTM-O365BootCamp-Planner101/SitePages/Course.aspx?CT=1614112471675&OR=OWA-NT&CID=bf1a6c31-71aa-959c-189c-5a11f7d416e2) (2 hour class) – Mar 9th

**Comment by amenity** [comment]
[Notion Enterprise](https://www.notion.so/pricing) might be an interesting solution to this.
![Screen Shot 2022-03-28 at 11 43 11 AM](https://user-images.githubusercontent.com/1463708/161142281-ad4f93d7-080f-4d74-9590-3b125fb21d51.png)
